corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -103,8 +103,8 @@ cloudCare.Details = Backbone.Model.extend({
 cloudCare.caseViewMixin = {
     lookupField: function (field) {
         var parentCase;
-        if (isParentField(field) && this.options.casedb) {
-            parentCase = this.options.casedb.get(this.model.get('indices').parent.case_id);
+        if (isParentField(field) && this.model.get('casedb')) {
+            parentCase = this.model.get('casedb').get(this.model.get('indices').parent.case_id);
```

Comment:
interesting. What happens if the index identifier is not `parent`? (I'm assuming that's what `.parent` is referring to).